src/pk/dh/dh_make_key.c

```diff
@@ -0,0 +1,192 @@
+/* LibTomCrypt, modular cryptographic library -- Tom St Denis
+ *
+ * LibTomCrypt is a library that provides various cryptographic
+ * algorithms in a highly modular and flexible manner.
+ *
+ * The library is free for all purposes without any express
+ * guarantee it works.
+ */
+
+#include "tomcrypt.h"
+
+#ifdef LTC_MDH
+
+static int dh_groupsize_to_keysize_internal(int groupsize)
```

Comment:
I still think it doesn't make sense to mark internal functions (be it static or 'library-internal') in some way and if you think you have to do it, please don't suffix static functions with `_internal` but prefix with `_` (we had that discussion already)